.circleci/config.yml

```diff
@@ -11,9 +11,7 @@ workflows:
           mvn-build-test-command: mvn clean verify -PbuildKar -Dit
           mvn-collect-artifacts-command: |
             mkdir -p ~/project/artifacts/junit/
-            cp ~/project/nexus-repository-p2/target/surefire-reports/*.xml ~/project/artifacts/junit/
-            cp ~/project/nexus-repository-p2/target/nexus-repository-* ~/project/artifacts/
```

Comment:
Minor issue: you removed this line and it won't be found by your `find` below, since that's just for xml files in surefire-reports.

Reply:
Do we need jars during build-and-test workflow? As I can see in R - no? https://github.com/sonatype-nexus-community/nexus-repository-r/blob/master/.circleci/config.yml

Reply:
One issue (I think Jeffry was pointing to) with the single "find" command is it will not capture the output of the integration tests. As an example, other projects use the following additional line to capture IT output as a CI artifact:

```
cp ~/project/nexus-repository-p2-it/target/failsafe-reports/*.xml ~/project/artifacts/junit/
```

There may be a "more portable" (e.g. less project specific) incantation as well.

Reply:
Ok, I return this line